source/getting-started/user-guide/amds-interface.md

```diff
@@ -89,85 +89,84 @@ Place the code below into your custom user app init function. Modify the `GP3IO_
 gp3io_mux_set_device(GP3IO_MUX_2_BASE_ADDR, GP3IO_MUX_DEVICE1);
 ```
 
-## Trigger ADC Sampling
+## Requesting and Retrieving Data from the AMDS
 
-The AMDC firmware drivers trigger AMDS sensor sampling synchronous to both the peak and valley of the PWM carrier triangle wave.
-When AMDS support is enabled, the ADC sampling is automatically enabled.
-If needed, the ADC sampling can be disabled via the `mb <idx> adc <on|off>` command or the associated C code API.
+Triggering data acquisition on the AMDS is done via the AMDC Timing Manager. After enabling the AMDS (on the correct port) via the Timing Manager, the AMDS will be included when all enabled sensors are triggered by the Timing Manager. Sensors are triggered in sync with the peaks and/or valleys of the PWM carrier. To learn more about how to enable sensors and how to configure triggering, please read the documentation for the [Timing Manager](/firmware/arch/systems.md).
 
-## Request Data Transmission from AMDS to AMDC
+After the AMDS is triggered to begin sampling, each sensor card populated on the AMDS will sample and provide the data the the processor on the mainboard. As soon as the mainboard has collected all the data from the sensor cards, the mainboard will automatically send all the data back to the AMDC, where the data for each sensor card will be made available in the corresponding channel's data register. To learn more about the firmware interface between the AMDC and AMDS please see the [AMDS Firmware documentation](/accessories/amds/firmware/index.md), and the AMDC driver code ([FPGA code](https://github.com/Severson-Group/AMDC-Firmware/tree/develop/ip_repo/amdc_amds_1.0) and [C driver](https://github.com/Severson-Group/AMDC-Firmware/blob/develop/sdk/app_cpu1/common/drv/amds.c)). 
```

Comment:
> the the

Reply:
I never like linking to the code tree in github, since these might change later.... maybe just provide the path as `ip_repo/amdc_amds_1.0` for fpga and `drv/amds` for c code...?